src/docs/Contributing/Principles/ContributionJourney/index.mdx

```diff
@@ -0,0 +1,121 @@
+
+### ❓ What?
+
+The design phase is a time consuming part of the contribution. For that reason, the designer will have to work on its own for a little time to do the research it's needed. The developer doesn't need to be available during this period other than just answering a few questions from the designer if he has some.
+
+<Highlight theme="tips" title="">
+
+💡 The developer can take advantage of this period to install the project on his environment and try to understand how the files and the documentation is structured. Feel free to contact Mozaic's tech team on Slack if you need any kind of support.
+
+</Highlight>
+
+### 👩‍🚀 Who?
+
+- Designer
+- Mozaic's UI support
+
+### 🎬 How?
+
+The designer working on this particular pattern will definitely need to pair with a Mozaic's UI.
+
+### ⏲ How long?
+
+This task can take up to a week to be done well for some complex patterns.
+
+### 🚀 Deliverables
+
+- First version of the Sketch files
+
+# 3. FIRST POOLING
+
+### ❓ What?
+
+This part is very important because this is when the designer will submit his work to the developer. The developer will evaluate if the design work is technically doable. The other part of the pooling will allow them to talk about the first documentation draft. Is it still relevant and does it mean something for the end user ?
+
+### 👩‍🚀 Who?
+
+The duo will also have to see if the designed pattern responds to digital standards (accessibility, UX behaviour, etc.). They can ask for Mozaic's team and LMF core teams.
+
+### 🎬 How?
+
+The duo will be exchanging together to discuss technical and design issues **together**.
+
+### 🚀 Deliverables
+
+- Technical validation
+- Documentation summary validated
+
+For the designer only, **Abstract** collections with:
+    - Spacings specs
+    - Board with different variations (sizes, colors, states, etc.)
+
+# 4. DEVELOPMENT
+
+### ❓ What?
+
+The development phase is very crucial. This is the time that will make the pattern available to our users. The developer should be able to start working on coding the pattern because the project should be installed on his environment at this point.
+
+### 👩‍🚀 Who?
+
+The developer helped by Mozaic's tech team and LMF core team.
+
+### 🗓 When?
+
+This development phase is only efficient when it's done after the first pooling validating the design of the pattern.
+
+### 🚀 Deliverables
+
+- Developed patterns
+- Code documentation done by the developer
+- Specs documentation page done by the developer
+
+# 5. SECOND POOLING
+
+### ❓What & 👩‍🚀 Who?
+
+This second pooling is the time when the designer and the developer need to sit together to validate the pattern. Communicating during the whole process is very important but this time it's a necessity.
+
+### 🎬 How?
+
+At this point, the duo will review the whole pattern to deliver the design, the code and the documentation at once. The designer provides his documentation to the developer. Next, the developer puts the whole duo's work on GitHub.
```

Comment:
Maybe we can be more precise and say that the designer provides his documentation in a certain format (word, pdf, google doc, and so on.) and it's up to the developer to transform it into Markdown files.